src/Features/CSharp/Portable/ImplementInterface/CSharpImplementInterfaceService.cs

```diff
@@ -67,85 +66,29 @@ public CSharpImplementInterfaceService()
 
         protected override bool HasHiddenExplicitImplementation => true;
 
-        private static ClassDeclarationSyntax GetClassDeclarationAt(SyntaxNode root, int position)
+        protected override SyntaxNode AddCommentInsideIfStatement(SyntaxNode ifStatement, SyntaxTriviaList trivia)
         {
-            var node = root.FindToken(position).Parent.FirstAncestorOrSelf((SyntaxNode n) => n.IsKind(SyntaxKind.ClassDeclaration));
-            return node as ClassDeclarationSyntax;
+            return ifStatement.ReplaceToken(
+                ifStatement.GetLastToken(),
+                ifStatement.GetLastToken().WithPrependedLeadingTrivia(trivia));
         }
```

Comment:
i couldn't find a language agnostic way to do this.     So VB and C# both have  way to add a comment inside their if-block.